dialect/sql/sqlgraph/graph.go

```diff
@@ -111,6 +117,22 @@ func To(table, column string) StepOption {
 	}
 }
 
+// ToSchema sets the db name of the From
+// step
+func ToSchema(name string) StepOption {
+	return func(s *Step) {
+		s.To.Schema = name
+	}
+}
+
+// EdgeSchema sets the db name of the EDge
+// table
+func EdgeSchema(name string) StepOption {
+	return func(s *Step) {
+		s.Edge.Schema = name
+	}
+}
+
```

Comment:
Let's start without these options for now, and just change the actual `Step` in the tests. For example:
```go
input: func() *Step {
	step :=  NewStep(
		From("nodes", "id", 1),
		To("nodes", "id"),
		Edge(O2O, true, "nodes", "prev_id"),
	),
	step.To.Schema = "db1"
	return step
}()
```

I want to see how we're going to use it in the codegen, because I don't really like these options (prefer to just change the actual ones).

Reply:
Sounds good, just FYI this is what it looks like so far: https://github.com/marwan-at-work/ent/compare/schemagraph...marwan-at-work:schematemplate?expand=1

Reply:
Let's re-added them after working on the codegen. I still think on other alternative. Sounds good?

Reply:
@a8m i already have the tests adjusted to the alternative, so i can just push that up :)